projects/plugins/jetpack/composer.json

```diff
@@ -37,7 +37,8 @@
 		"automattic/jetpack-sync": "1.21.x-dev",
 		"automattic/jetpack-terms-of-service": "1.9.x-dev",
 		"automattic/jetpack-tracking": "1.13.x-dev",
-		"nojimage/twitter-text-php": "3.1.2"
+		"nojimage/twitter-text-php": "3.1.1",
+		"automattic/jetpack-identity-crisis": "@dev"
```

Comment:
Do you think you could keep that list ordered alphabetically, and use a version other than `@dev`? Maybe `0.1.x-dev` is what's appropriate here until we do a first release?

Reply:
Also watch the, I assume, unintentional downgrade of the twitter text package.